OneCommunityGlobal/HighestGoodNetworkApp
# Ramya Lint fix for ManageMemberships Component

# Description
Removed the component from the project as it is not used anywhere


## Main changes explained:
- No major changes
-  Removed the component  from .eslintignore and .prettierignore files
- Deleted the component from project directory

## How to test:

1. check into current branch
2. do npm install and ... to run this PR locally
3. Verify if the application works as expected
